### PR TITLE
fix!: editorial review

### PR DESCRIPTION
**Pull request by mfw78**
This PR:

1. Has multiple editorial corrections to converge on standardized nomenclature / terminology.
2. Fixes errors in a code example in README.md.
3. Fixes a typographical error for exported interface types (reason for breaking change, per semver).

### CODE REVIEW

**Comment by mfw78** [comment]
I have read the CLA Document and I hereby sign the CLA

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: gnosis/safe-react-eslint-plus-action@v4.0.0
+      - uses: Maggi64/eslint-plus-action@master
```
Comment (anxolin) [inline .github/workflows/lint.yml:9]: not sure why we had safe-react there, but also why this change in this PR?

**Comment by mfw78** [comment]
@anxolin Can we kindly get another review so this can be merged? I have a moderately large PR forth-coming for merging in `ComposableCoW` support and want to avoid rebases.

**Review by alfetopito — APPROVED** [review]

File: .github/workflows/lint.yml (hunk 1 above)
Comment (anxolin) [reply .github/workflows/lint.yml:9]: @mfw78 why did we need to change this? was there something broken?

**Review by anxolin — APPROVED** [review]
